woocommerce/storefront
# Remove unnecessary WP dependencies

I think we should be fine removing `@wordpress/prettier-config` and `@wordpress/stylelint-config` from the list of dependencies given that they are included in `@wordpress/scripts`.

### How to test the changes in this Pull Request:
1. Run `npm ci` to reinstall npm packages according to the new `package-lock.json`.
2. Run `npm run build` and smoke test your store to make sure CSS and JS files were built correctly.
3. Run `npm run lint:css` to verify CSS linting keeps working fine.

### Changelog

> Fix – Removed unnecessary dependencies.
